rust/tests/read_delta_log_test.rs

```diff
@@ -150,6 +150,15 @@ async fn test_read_liquid_table() -> DeltaResult<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn test_read_table_features() -> DeltaResult<()> {
+    let path = "./tests/data/table_with_liquid_clustering";
+    let mut _table = deltalake::open_table(&path).await?;
+    assert!(_table.get_state().reader_features().is_some());
+    assert!(_table.get_state().writer_features().is_some());
```

Comment:
We should be able to assert the specific features here, right?

Reply:
Yes, I have more/better tests I'm going to check in prolly tomorrow.